error: no template named 'unordered_map' in namespace 'std'

I'm getting the error below when I try to compile:
```
$ make
awk -f glyph.awk glyph.def > glyph.inl
c++ -std=c++17 -Wall -Wextra -Os -MD -MP -MF cxxmatrix.dep -c -o cxxmatrix.o cxxmatrix.cpp
cxxmatrix.cpp:597:17: error: no template named 'unordered_map' in namespace 'std'
    static std::unordered_map<char32_t, glyph_definition_t const*> map;
           ~~~~~^
1 error generated.
make: *** [cxxmatrix.o] Error 1
```

I'll create a PR shortly.

--- a/cxxmatrix.cpp
+++ b/cxxmatrix.cpp
@@ -14,6 +14,7 @@
 #include <vector>
 #include <algorithm>
 #include <iterator>
+#include <unordered_map>
 
 #include "cxxmatrix.hpp"
 #include "mandel.hpp"

